sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -13,6 +13,13 @@
 
 ### Other Changes
 - Improved error messages for the `evaluate` API by enhancing the validation of input parameters. This update provides more detailed and actionable error descriptions.
+- To align with our support of a diverse set of models, the following evaluators will now how have a new key in their result output without the `gpt_` prefix. To maintain backwards compatibility, the old key will with the `gpt_` prefix will still be present in the output; however, it is recommended to use the new key moving forward as the old key will be deprecated in the future.
```

Comment:
If I understand correctly, both keys (w/ and wo/ prefix) will be returned with this change. Will it cause problems on UI, especially the detail view and compare view? Now user can see duplicated metrics on UI unless UI add some filtering logic to dedup.

Reply:
I understand that renaming might impact existing customers, but I doubt they would directly rely on those keys in their code today. I could be wrong, but I believe the typical use case is that customers upload the evaluation results to AI Studio and view them there.

That being said, maybe it's still worth re-considering the rename as an option since we could make this change before GA. UI side, they need support new keys, and keep old keys for backword compatibility anyways.

Reply:
No - UI was consulted on this and we are working with them. They will only show one.